frontend/src/components/company/CompanyDetails.tsx

```diff
@@ -0,0 +1,133 @@
+              </Typography>
+              <div className='companyType'>
+                <Typography variant='h4' component='h2'>
+                  {selectedCompany.location}
+                </Typography>
+              </div>
+            </div>
+          </div>
+        </div>
+
+        <div className={style.companyDetailMainContainer}>
+          <div className={style.aboutCompanyContainer}>
+            <div className={style.jobSubhead}>
+              <Typography variant='body1' component='h2'>
+                About Company
+              </Typography>
+            </div>
+            <Typography variant='h4' component='h2'>
+              {selectedCompany.about}
+            </Typography>
+          </div>
+
+          <div className='skill-container'>
+            <Typography variant='body1' component='h2'>
+              Services
+            </Typography>
+            <ul className='skillsList'>
+              {selectedCompany.jobs.slice(0, 3).map((job) => (
+                <li key={job.id}>{job.skills}</li>
+              ))}
+            </ul>
+          </div>
+
+          <div className={style.jobDescription}>
+            <div className={style.jobSubhead}>
+              <Typography variant='body1' component='h2'>
+                Available Positions
+              </Typography>
+            </div>
+            <div className={style.jobListContainer}>
+              {selectedCompany.jobs.map((job) => (
+                <div key={job.id} className={style.jobList}>
+                  <div className={style.jobName}>
+                    <div>
+                      <Typography variant='body1' component='h2'>
+                        {job.title}
+                      </Typography>
+                    </div>
+                    <div>
+                      <Typography variant='h4' component='h2'>
+                        {job.experience} years Experience
+                      </Typography>
+                    </div>
+                  </div>
+                  <div className={style.jobDescription}>
+                    <Typography variant='h4' component='p'>
+                      {job.description}
+                    </Typography>
+                  </div>
+                </div>
+              ))}
+              <hr></hr>
+            </div>
+          </div>
+
+          <div className={style.companyAddress}>
+            <div className={style.jobSubhead}>
+              <Typography variant='body1' component='h2'>
+                Office Address
+              </Typography>
+              {selectedCompany.location}
+            </div>
+            <Typography variant='h4' component='h2'>
+              <div className={style.map}>
+                <Map key={selectedCompany.id} city={selectedCompany.location} />
+              </div>
+            </Typography>
+          </div>
+        </div>
+      </div>
+      <div className={style.rightContainer}>
+        {/* <div className='job-card-container'>
```

Comment:
still needed?

Reply:
Hi Petr, Yes Sayali and Remta working on it. commit to be expected later tomorrow  :grin: